deduplicator/src/lib/db_hashes.rs

```diff
@@ -15,16 +15,16 @@ const TABLE_ADDRESSES: &str = "addresses";
 /// Name of the table containing hashes. For each address, multiple hashes may be stored.
 const TABLE_HASHES: &str = "_addresses_hashes";
 
-/// Name of the table listing addresses that have to be removed to eliminate all duplicates.
-const TABLE_TO_DELETE: &str = "_to_delete";
-
 /// A database, this structure can be used to open connections or perform high-level operations.
 pub struct DbHashes {
     db_path: PathBuf,
+    to_delete: HashSet<i64>,
```

Comment:
IMO it could be difficult to understand that the hashset is outside the sqlite DB, because sometimes it could be weird to manipulate the Hashset from the DbHashes instance. I don't have a lot of idea apart from renaming the struct such as `HashesStorages`